src/components/Input.js

```diff
@@ -4,34 +4,163 @@ import {
   FormHelperText,
   InputAdornment,
   InputLabel,
-  Input as MuiInput,
+  OutlinedInput,
 } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import { IconCheck } from '~/styles/icons';
-
 const useStyles = makeStyles((theme) => ({
   inputLabel: {
+    transform: 'translate(28px, -6px)',
+    background: theme.custom.colors.white,
+    fontSize: '12px',
+    padding: '2px 6px',
+    transition: 'none',
+    color: (isOrganization) =>
+      isOrganization
```

Comment:
these things are repeated a lot. Please define constants for color based on organization and DRY :)